handbook/company/pricing-features-table.yml

```diff
@@ -334,14 +359,23 @@
       tier: Premium
 - categoryName: Device management
   features:
-    - industryName: User-initiated enrollment of macOS computers
+    - industryName: Interactive MDM migration #   « end-user initiated MDM migration, with interactive UI
```

Comment:
@noahtalerman MDM migration should be free

Reply:
Agreed. Right now it's paid because it only works for DEP Macs. We did this to move quickly for our large gaming company customer.